proposals/local_regional_mentor_communities.md

```diff
@@ -0,0 +1,45 @@
+### Prospectus for Local/Regional Mentor/Community groups
+
+#### Goals
+
+The group will facilitate and help those Carpentry members and Member Organizations who wish to  organize and maintain local mentoring groups. This group will also discuss regional community building based on shared missions or topics matching the various interests of regional Carpentry members. 
+
+*Examples:*
+	
+**Local Mentoring Group**
+
+The University of Oklahoma (OU) has established a local, university based group that is made up of the locally trained Carpentry instructors that are part of the yearly cohort trained as part of the benefits of being a Member Organization. Allows for sharing of information and the ability to help new instructors with the certification process and help to integrate them into the workshop rotation of instructing at self-organized workshops. 
+
+**Regional Community **
+
+*Members: Univ of Oklahoma, Oklahoma State Univ., and Univ. of North Texas*
+A group of three university libraries are current participants of the Member Organizations. With shared interests of professional development of library staff based around the current development of the Library Carpentry, there is an opportunity to work together. Activities and meetings can be developed around various topics and tools that are found in the development environment found at research libraries that are common to the libraries involved.
+
+	
+
+#### Steps
+
+1. Establish regions with which to run this pilot. (US only? US/Europe? etc) (Jan 2017)
+1. Select/vet a volunteer group to serve as sub-committee members (Jan-Feb 2017)
+	1. First meeting of committee (end of Feb 2017)
+1. Establish a calendar of monthly community phone calls
+	1. one or two calls on the selected day?
+1. Create communication channels, archives and associated information tools and or methods (Feb 2017, after committee meets)
+1. Begin monthly calls.  (Mar 2017)
+1. Evaluate effectiveness of the group and determine continuing or termination of group after agreed upon time. 
```

Comment:
It's probably worth picking a time for this (Jan 2018?).  And the fixed evaluation point makes this seem like [a task force][1] that may transition into being a subcommittee.  Or maybe it's a subcommittee, and new steering committees will evaluate all subcomittees (somewhat related discussion in #144)?  I'd rather see this content PRed into `subcommittees/local-group` (or `TaskForces/local-group`) using the template (more details [here][2]).

[1]: https://github.com/swcarpentry/board/tree/48c8a1f47297e769ea14298e0f3d4182e0c07165/TaskForces#purpose-and-rationale
[2]: https://github.com/swcarpentry/board/blob/48c8a1f47297e769ea14298e0f3d4182e0c07165/subcommittees/proposal_instructions.md#by-pull-request